src/Sylius/Bundle/CartBundle/spec/EventListener/CartSubscriberSpec.php

```diff
@@ -102,29 +102,26 @@ function it_should_remove_a_item_to_a_cart_from_event(CartItemEvent $event, Cart
         $this->removeItem($event);
     }
 
-    function it_should_clear_a_cart_from_event(CartEvent $event, CartInterface $cart, $manager, $provider)
+    function it_should_clear_a_cart_from_event(CartEvent $event, CartInterface $cart, $manager)
     {
         $event->getCart()->willReturn($cart);
         $manager->remove($cart)->shouldBeCalled();
         $manager->flush()->shouldBeCalled();
-        $provider->abandonCart()->shouldBeCalled();
 
         $this->clearCart($event);
     }
 
-    function it_should_save_a_valid_cart($manager, $provider, CartEvent $event, CartInterface $cart)
+    function it_should_save_a_valid_cart($manager, CartEvent $event, CartInterface $cart)
```

Comment:
If you're changing this line, replace `$manager` with `ObjectManager $manager` :)